### ISSUE TITLE
Combine custom competition tabs with the existing sticky competition nav bar

### ISSUE THREAD

**Issue by jfly** [thewca/worldcubeassociation.org#1670]
@KitClement tells me that people find the current set of 2 navigations confusing:

![image](https://cloud.githubusercontent.com/assets/277474/26372967/9f431904-3fb4-11e7-86a5-2d61c80ca182.png)

He suggests moving the custom, horizontal tabs into the sticky navbar on the left. That would look something like this:

![image](https://cloud.githubusercontent.com/assets/277474/26373059/03eb227a-3fb5-11e7-9cac-c82db3d4b08b.png)

I suppose we'd leave the icons blank here, although it would be very sexy to allow custom icons as well =)

@jonatanklosko, I know you implemented the current tabs using bootstrap and javascript. Would it be possible to switch the tabs to a more boring, javascriptless solution?

**Comment by jonatanklosko** [comment]
It's certainly possible, though I'm not quite convinced if stacking more tabs within the left menu decreases the complexity. Also introducing *reloadingly-javascriptless version* seems like a step backwards for me.

**Comment by jfly** [comment]
About the complexity, @KitClement says:

> I've noticed people commenting that this is confusing to a first time user, and they often see one and not the other

I agree that the 2 navbars don't have any logical difference between each other, and that it's easy to look at one but not the other. I think that combining them is a decrease in complexity (but I'm no UI guy).

> Also introducing reloadingly-javascriptless version seems like a step backwards for me.

It does not feel that way to me. The less javascript we have to maintain, the easier our lives will be.

**Comment by KitClement** [comment]
Yeah, in my limited experiences with the new system, I get a few people who
register and then come to the competition and miss all of the information
in the tabs, or people who see only the tabs and then email us asking how
to register, as they missed the sidenav. Also, when I shared our first
competition using this system to our facebook group, one person kept asking
where the website was, noticing the registration system but not any of the
tabs.

**Comment by dunkOnIT** [comment]
Marked `could-close` as this will be addressed by the Slate redesign
